sktime/distances/_numba_utils.py

```diff
@@ -135,10 +137,10 @@ def to_numba_pairwise_timeseries(x: np.ndarray) -> np.ndarray:
     num_dims = _x.ndim
     if num_dims == 1:
         shape = _x.shape
-        _x = np.reshape(_x, (1, 1, shape[0]))
+        _x = np.reshape(_x, (shape[0], 1, 1))
     elif num_dims == 2:
         shape = _x.shape
-        _x = np.reshape(_x, (1, shape[1], shape[0]))
+        _x = np.reshape(_x, (shape[0], 1, shape[1]))
```

Comment:
if num_dims == 2 are we assuming this is (n,m), (d,m) or (m,d)? this seems to convert (n,m) to (n,1,m) and (d,m) to  (d,1,m)?